nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -52,7 +52,7 @@ in
       description = "Whether to enable seamless mode. When activated windows from the guest appear next to the windows of the host.";
     };
 
-    draganddrop = mkOption {
+    dragAndDrop = mkOption {
```

Comment:
This should have received a mkRenamedOption

Reply:
added #317756